sunbeam-microcluster/api/manifests.go

```diff
@@ -0,0 +1,91 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+
+	"github.com/canonical/lxd/lxd/response"
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/microcluster/rest"
+	"github.com/canonical/microcluster/state"
+	"github.com/gorilla/mux"
+
+	"github.com/openstack-snaps/snap-openstack/sunbeam-microcluster/api/types"
+	"github.com/openstack-snaps/snap-openstack/sunbeam-microcluster/sunbeam"
+)
+
+// /1.0/manifests endpoint.
+var manifestsCmd = rest.Endpoint{
+	Path: "manifests",
+
+	Get:  rest.EndpointAction{Handler: cmdManifestsGetAll, ProxyTarget: true},
+	Post: rest.EndpointAction{Handler: cmdManifestPost, ProxyTarget: true},
```

Comment:
```suggestion
	Post: rest.EndpointAction{Handler: cmdManifestsPost, ProxyTarget: true},
```
This one should stay plural

Reply:
Done